Paper.tex

```diff
@@ -755,7 +755,7 @@ \section{Message Call} \label{ch:call}
 0 & \text{if} \quad \boldsymbol{\sigma}^{**} = \varnothing \\
 g^{**} & \text{otherwise}
 \end{cases} \\
-(\boldsymbol{\sigma}^{**}, g^{**}, \mathbf{s}, \mathbf{o}) & \equiv & \begin{cases}
+\qquad (\boldsymbol{\sigma}^{**}, g^{**}, A, \mathbf{o}) & \equiv & \begin{cases}
```

Comment:
Does the `\qquad` make a difference here?  I thought the whole tuple is right-aligned toward the `\equiv` symbol.